subjects/blockchain/non-fungible-cats/audit/README.md

```diff
@@ -14,11 +14,11 @@
 
 ##### buy cat 1 at the listed price (10wei)
 
-###### The buy function functions properly
+###### The buy function works properly
 
 ##### transfer cat 1 to a secondary user
 
-###### The transfer function properly works
+###### The transfer function works properly
```

Comment:
It might be better to rephrase this as a yes/no question. For example

Does the transfer function work properly?

Reply:
All must be phrased as questions, with "yes" as the happy path.

Reply:
done